src/seedu/addressbook/commands/attendance/ViewAttendanceDateCommand.java

```diff
@@ -0,0 +1,77 @@
+package seedu.addressbook.commands.attendance;
+
+import static seedu.addressbook.common.Messages.MESSAGE_DATE_CONSTRAINTS;
+import static seedu.addressbook.common.Utils.isValidDate;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+
+import seedu.addressbook.commands.Command;
+import seedu.addressbook.commands.commandresult.CommandResult;
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.details.Name;
+
+/**
+ *  Lists down the people who were present on the particular date.
+ */
+public class ViewAttendanceDateCommand extends Command {
+
+    public static final String COMMAND_WORD = "viewAttenDate";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n"
+            + "Views the attendance of the date. \n"
+            + "Parameters: d/date \n"
+            + "\tExample: " + COMMAND_WORD + " " + "d/28-10-2018";
+    public static final String MESSAGE_SUCCESS = "Attendance for the given date, ";
+
+    private String date;
+
+    // Constructor
+    public ViewAttendanceDateCommand(String date) throws IllegalValueException {
+        if (!isValidDate(date)) {
+            throw new IllegalValueException(MESSAGE_DATE_CONSTRAINTS);
+        }
+        this.date = date;
+    }
+
+    /**
+     * Constructor used for Privileges
+     * Command constructed has no functionality
+     * */
+    public ViewAttendanceDateCommand() {
+        // Does nothing
+    }
+
+    @Override
+    public CommandResult execute() {
+        try {
+            String outputDate = date;
+            String present = "";
+            String absent = "";
+            if ("0".equals(date)) {
+                outputDate = new SimpleDateFormat("dd-MM-yyyy").format(new Date());
```

Comment:
Use DateTimeFormatter instead as it is safer. eg.
outputDate = LocalDate.now().format(DateTimeFormatter.ofPattern("yyyy-MM-dd")

Reply:
why yyyy-MM-dd?

Reply:
oh. dd-MM-yyyy then.